src/projects/shared/site/src/containers/MainDesigner/index.tsx

```diff
@@ -46,6 +46,10 @@ export const MainDesigner = ({ otherPlace }: Props) => {
         dispatch(SetCircuitSaved(false))),
     [designer]);
 
+    useEffect(() => {
+        designer.viewport.margin = { top: HEADER_HEIGHT };
```

Comment:
Do this in the useLayoutEffect when setting `window.Circuit` or maybe just even in the root `index.tsx` when creating the designer?

Reply:
Unless I missed it, index.tsx is in digital and I think this logic could be in shared. I'll put it in the other useLayoutEffect then